itechart_project/settings.py

```diff
@@ -145,7 +145,7 @@
 # STATIC_ROOT = os.path.join(BASE_DIR, 'static')
 STATICFILES_DIRS = (os.path.join(BASE_DIR, 'static'),)
 
-IGDB_API_KEY = os.getenv('IGDB_API_KEY')
+IGDB_API_KEY = os.environ.get('IGDB_API_KEY')
```

Comment:
I think we can rewrite all env variables imports in following way(and let's do it):
```
def _get_env_variable(variable_name):
    try:
        return os.environ[variable_name]
    except KeyError:
        error_msg = f'Set the {variable_name} environment variable'
        raise EnvironmentError(error_msg)

IGDB_API_KEY = _get_env_variable('IGDB_API_KEY')
```
In this way we can see error **before** app starts with incorrect settings